### PR TITLE
Add webpage for the workshop

### PR DESCRIPTION
**Pull request by Miffyli**
A preliminary webpage to describe the workshop for the competitions. Would be nice to have merged ASAP so can share around :)

### CODE REVIEW

File: workshop/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: Getting Started - Workhshop
+---
+
+## August 5th 2021, virtual ([gather.town](https://gather.town/))
```
Comment (brandonhoughton) [inline workshop/index.md:6]: Link generic (assuming for now)

File: workshop/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+
+<div class="sidebarstatus">
+    <iframe src="https://discordapp.com/widget?id=565639094860775436&theme=dark" width="100%" height="400" allowtransparency="true" frameborder="0"></iframe>
+</div>
+
+Are you interested in creating bots for Minecraft, or even competing in the [Diamond](/diamond) or [BASALT](/basalt) competitions, but do not know where to start?
+**Then this is the event for you!**
```
Comment (brandonhoughton) [inline workshop/index.md:14]: Woo! =)

File: workshop/index.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+<h3 style="width: 100%; text-align: center;">Prerequisites</h3>
+
+We wish to assist you regardless of your proficiency in programming or machine learning, but that being said, we ask that you at least...
+
+* have a stable internet connection to communicate with us and to run MineRL environment, either locally or on [Google Colab](https://research.google.com/colaboratory/)
+* have basic Python knowledge. Read through *at least* three first chapters of [this tutorial](https://docs.python.org/3/tutorial/). Go through the first few Colab example notebooks if you intend to use it.
+* have tried installing the [MineRL environment](https://github.com/minerllabs/minerl) on your machine. Do not fear if it fails! You can still use Colab to run MineRL! 
+* bring a microphone and a webcam with you for gather.town. This is not a hard requirement, but it will make communication much more fluent! Be prepared to share your screen with tutors, in case there are some code bits you need help with.
```
Comment (brandonhoughton) [inline workshop/index.md:32]: +1 for mic =)

**Review by brandonhoughton — APPROVED** [review]
Looks great!

File: workshop/index.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Join us during our workshop to complete tasks specifically designed to ease the entry into the AI development in Minecraft! The best part? We (the organizers) will be present to tutor and help you out! We hope that every participant leaves the workshop with freshly gained skills to work towards better AI with the [MineRL environment](/index)!
+
+We realize that you, our dear reader, might be well versed in the runes of Python as is, or maybe you already master the arcane dark arts of machine learning. Considering this,
+we will target the tasks towards three groups:
+
+* *Newcomer*: you are new to programming and the MineRL environment (maybe familiar with Minecraft). We are happy to get you started by making simple Minecraft bots, but please take care to complete the steps listed below in prerequisites!
+* *Programmer*: you are familiar with programming. Your adventure will start with getting MineRL environment running and maybe a touch of machine learning!
+* *Machine learner*: you are already familiar with machine learning (e.g. computer vision, reinforcement learning, etc), and are keen to try out MineRL. We will prep you to start competing in the competitions right away!
```
Comment (shwang) [inline workshop/index.md:23]: Machine learner 😂

**Pull request merged**